src/openforms/analytics_tools/tests/test_csp_update.py

```diff
@@ -15,10 +16,33 @@ def parse_csp_policy(header_value):
     return csp_values
 
 
-@override_settings(
-    CSP_REPORT_ONLY=False,
-)
+@override_settings(CSP_REPORT_ONLY=False, SOLO_CACHE=None)
 class CSPUpdateTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        config = AnalyticsToolsConfiguration.get_solo()
```

Comment:
if possible, we prefer mocking django-solo models so that you don't have to deal with cache and cross-contamination in tests